Form-Controls/index.html

```diff
@@ -9,19 +9,53 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+
   <header>
-    <h1>Product Pick</h1>
+    <h1>T-shirt order form</h1>
+    <h2 id="heading">choose your t-shirt here!</h2>
   </header>
+
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+
+  <form method="post" class="form">
+      <label for="name" ><h2>Name:</h2></label>
+      <input type="text" id="name" name="user_name" minlength="2"  maxlength="20" placeholder="Enter your name here" required/>
```

Comment:
Is a maxlength a useful constraint? What would happen if someone had a name longer than 20 characters?